### ISSUE TITLE
Make frame-title-unique a reviewOnly WCAG412 rule

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#3487]
Aze-core's `frame-title-unique` rule is currently inconsistent with ACT Rules because it is far too strict. A better approach to this rule would be to change the tags, give it the `wcag412` and `wcag2a` tags, and remove `best-practice`, then add `"reviewOnFail": true` to the rule so it never flags things as violations. We simply cannot tell these are WCAG violations. This makes the rule partially consistent with the [Iframe elements with identical accessible names have equivalent purpose](https://wai-wcag-act-rules.netlify.app/standards-guidelines/act/rules/4b1c6c/proposed/) ACT rule.

**Comment by padmavemulapati** [comment]
Validated in the latest develop branch code base, 
`frame-title-unique` is showing in the `incomplete` results now and also having no `best-practice` tag value.
We are seeing only tags, 
```
0: "cat.text-alternatives"
1: "wcag412"
2: "wcag2a"
```
![image](https://user-images.githubusercontent.com/47409579/187217878-a94ef4ae-7260-4079-9f17-b663437e2bdc.png)

Earlier this rule is in list of `violations`, and also having only `best-practice` tag 

![image](https://user-images.githubusercontent.com/47409579/187218080-c573365d-9bb4-40a0-a1f9-703aa618b4c0.png)
